packages/theme-data/src/colorSchemes/lightGray/components/slider.js

```diff
@@ -0,0 +1,25 @@
+export default {
```

Comment:
I'm wondering if most of this stuff can be moved into the `baseTheme` file since that maps to the `light-grey` theme. Aslo `basics.colors.charcoal600` can map to the `colorScheme`.